scripts/run_aws_pipeline.sh

```diff
@@ -5,7 +5,7 @@
 scene=""
 burst_ids=()
 resolution=20
-output_crs=""
+output_crs="UTM"
```

Comment:
Ah, I see that you now have this as the default here... in which case, it might be fine to leave the default as None in the cli.